.github/workflows/clojure.yml

```diff
@@ -146,10 +146,11 @@ jobs:
       working-directory: server
       run: clojure -M:lint "--config" ".clj-kondo/ci-config.edn"
 
-  publish:
+  publish-image:
```

Comment:
Small change to make the deploys faster, we build the docker image while running the tests, but only create the eb application version if the tests pass.